index.html

```diff
@@ -2374,21 +2394,6 @@ <h3>
 consumption rules</a> in the [[INFRA]] specification.
         </p>
 
```

Comment:
@selfissued @msporny would this be acceptable?
```suggestion
        </p>
        <p class="note" title="@context">
"Please note that the value of the @context object member will be ignored, if present."
        </p>

```

Reply:
It reintroduces the problematic language. Here's another option that could go into this PR and then it can be further tweaked by editors after merge:

```suggestion
        </p>
        <p class="note" title="@context">
"Please note that the value of the `@context` property is JSON-LD specific representation syntax and must be treated according to the generic representation production and consumption rules."
        </p>
```